### PR TITLE
docs: add contribution guidelines

### LINKED ISSUE #21: contribution guidelines
We need some contribution guidelines and a style guide. Will fix this soon.

### PR DESCRIPTION
**Pull request by ghost**
Enforce a standard set of rules for contributors.

closes #21

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+
+Pull requests won't be merged when test coverage decreases. This helps ensure the quality of the project. To check coverage before submitting a pull request:
+
+```bash
+$ COVERAGE_FILE=.coverage_ssh coverage run tests/test_ssh.py
+$ COVERAGE_FILE=.coverage_validations coverage run tests/test_validations.py
+$ coverage combine .coverage_ssh .coverage_validations
+```
+
+### Code style rules
```
Comment (dvopsway) [inline CONTRIBUTING.md:29]: Do we need this whole section, Looks like a lot. A simple flake8 check will ensure if everything is good or not.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+    * NOTE(rjrhaverkamp): remeber, 1+1=2
+    * PERF(rjrhaverkamp): blazingly fast!
+* When catching exceptions, name the variable `ex`.
+* Use whitespace to separate logical blocks of code and to improve readability.
+* No single-character variable names except for trivial indexes when looping,
+or in mathematical expressions implementing well-known formulas.
+* Heavily document code that is especially complex and/or clever.
+* When in doubt, optimize for readability.
+
+### Commit Message Format
```
Comment (dvopsway) [inline CONTRIBUTING.md:54]: This is brilliant stuff.

**Review by dvopsway — COMMENTED** [review]
Overall looks good, We should reference this in README.md though.

File: CONTRIBUTING.md (hunk 2 above)
Comment (ghost) [reply CONTRIBUTING.md:54]: Great! I took most of it from https://github.com/falconry/falcon/blob/master/CONTRIBUTING.md though. :)

**Comment by ghost** [comment]
@dvopsway yes, you are right, I will fix that!

**Comment by ghost** [comment]
@dvopsway fixed in #23

**Pull request merged**